### PR TITLE
Add hook to prohibit yml files in apidoc

### PR DESCRIPTION
**Pull request by ls-samer-abuthaher**

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -328,6 +328,11 @@
     - openapi
   language: script
   files: '^apidoc/.+/openapi\.yaml$'
+- id: openapi-yaml-not-yml
+  name: Make sure openapi definitions end with .yaml
+  entry: It's YAML not YML
+  language: fail
+  files: '^apidoc/.*\.yml$'
```
Comment (ls-jad-elkik) [inline .pre-commit-hooks.yaml:335]: `.*` is recursive?

Comment (ls-jad-elkik) [reply .pre-commit-hooks.yaml:335]: Ah it's regex

Comment (ls-jad-elkik) [reply .pre-commit-hooks.yaml:335]: ok bye

**Review by ls-jad-elkik — APPROVED** [review]

File: .pre-commit-hooks.yaml (hunk 1 above)
Comment (ls-samer-abuthaher) [reply .pre-commit-hooks.yaml:335]: it's regex no? so it will match `/`

**Comment by ls-samer-abuthaher** [comment]
it's leviosa not levioSAR:
```
samer.abuthaher ~/dev/reporting-service:assing-more-disk-emptydir lsksandbox:sbx-qaenv
% pre-commit run --all openapi-yaml-not-yml
[INFO] Initializing environment for https://github.com/lightspeed-hospitality/pre-commit-hooks.
Make sure openapi definitions end with .yaml.............................Failed
- hook id: openapi-yaml-not-yml
- exit code: 1

It's YAML not YML

apidoc/reporting-service/openapi-dynamic.yml
```

**Pull request merged**